### DISCUSSION TITLE
Tự học lập trình

### DISCUSSION THREAD

**Discussion by ghost** [codingreshapefuture/Community#31] [Q&A] [answered]
Mình có một phân vân là khi tự học lập trình thì mình nên học theo trình tự, học hết các bài giảng về C++ sau đó mới qua học cái khác như là Java, C#, các thứ về Front-end. Mình có nên học xen kẽ không: ví dụ : 2-4-6 học C++, 3-5-7 học về Java...

**Comment by lucthienphong1120** [comment, marked as answer]
Học Javascript sau khi học xong C++ là đẹp bạn. 
HTML, CSS ,Javascript, React, TypeScript kiểu gì bạn cũng phải học lúc làm Front-end à.
